benchmarks/hash_table/static_map/erase_bench.cu

```diff
@@ -59,12 +59,12 @@ std::enable_if_t<(sizeof(Key) == sizeof(Value)), void> static_map_erase(
   state.exec(
     nvbench::exec_tag::sync | nvbench::exec_tag::timer, [&](nvbench::launch& launch, auto& timer) {
       // static map with erase support
-      cuco::static_map<Key, Value> map{
+      auto map = cuco::experimental::static_map{
         size, cuco::empty_key<Key>{-1}, cuco::empty_value<Value>{-1}, cuco::erased_key<Key>{-2}};
-      map.insert(pairs.begin(), pairs.end(), {}, {}, launch.get_stream());
+      map.insert(pairs.begin(), pairs.end(), {launch.get_stream()});
 
       timer.start();
-      map.erase(keys.begin(), keys.end(), {}, {}, launch.get_stream());
+      map.erase_async(keys.begin(), keys.end(), {launch.get_stream()});
```

Comment:
I'm not that firm on how nvbench works when mixing streamed execution with custom timers, but since `erase_async` no longer syncs the stream internally, do we still need to use `nvbench::exec_tag::sync` (which adds a lot of noise)?

Reply:
> do we still need to use nvbench::exec_tag::sync (which adds a lot of noise)?



good point. let me remove it.